cli/command/image/build.go

```diff
@@ -344,9 +360,18 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 	// Everything worked so if -q was provided the output from the daemon
 	// should be just the image ID and we'll print that to stdout.
 	if options.quiet {
-		fmt.Fprintf(dockerCli.Out(), "%s", buildBuff)
+		imageID = fmt.Sprintf("%s", buildBuff)
+		fmt.Fprintf(dockerCli.Out(), "%s", imageID)
```

Comment:
That form tweaks my format string security vulnerability nerve, but maybe that's a vestige of a C background. Is passing a potentially untrusted string to a thing which expects a format string always safe in go? (I'd be inclined to go the obviously safe route nonetheless).